### ISSUE TITLE
Ticket description shouldn't appear in "follow-ups" section?

### ISSUE THREAD

**Issue by adrianboyko** [rossp/django-helpdesk#421]
It seems that the ticket description is always the first follow-up and this bothers me for two reasons... First, the original description is _not_ a follow-up. Second, the needless redundancy can be annoying, especially when the description is long as tends to be the case on my site.

**Comment by gwasser** [comment]
My guess is that this needs to be done as-is because there's no way to attach a file to the ticket description. If an incoming email generates a new report, and that email has an attachment, it would be lost unless it is attached to a FollowUp. Maybe it could leave out the FollowUp unless an attachment is detected (and even then, only add the attachment to a blank FollowUp)? I do agree that reducing redundancy is a good goal.

**Comment by rossp** [comment]
In hindsight, I think `FollowUp` is a poor naming choice here. 

If you think of each `Ticket` as a _conversation_ heading from request through to resolution, the initial report is the first part of the discussion thread. It is then followed by a series of follow-ups, comments, questions, and clarifications. In this context, it makes sense (to me at least 😄) that the `Ticket` `description` has equal prominence to the `FollowUp` items that follow.

Do you agree? If so, should we rename `FollowUp` (at least in the UI) and remove the `description` from the ticket display?

I'm not saying I'm right - this is just my line of thinking - and I'm open to further thoughts :)

**Comment by adrianboyko** [comment]
Removing description from the ticket display and renaming follow-ups in the UI works for me! I agree that it's good to keep the entire conversation together.

**Comment by gwasser** [comment]
I don't think `Ticket` `description` is used anywhere else, is it? I like keeping conversation together and removing `description`.

**Comment by Pauldic** [comment]
I agree with @adrianboyko's suggestion to remain Follow-ups to Discussion. I initially have challenge with that choice of word but ignored it

**Comment by reduxionist** [comment]
Thank you for the input @adrianboyko and @Pauldic. A change is clearly needed, I just wish we could float the options out to a larger audience -- where's that django-polls app when you need it? 😉 

I'll try out the replacement suggested here on my local guinea pigs and gauge their reaction, and collect any other possibilities that come up here in this ticket. It might not happen before 0.2.0 gets sent off to press at PyPI, but, barring any sudden revolt by current users, should be feasible for 0.2.1.

**Comment by gurupratap-matharu** [comment]
@DavidVadnais @uhurusurfa 

Based on the recent work on Ticket description page the ticket description **does** appear again in followups.
But I think the fix is really simple. 

This conversation also talks about renaming Followup -> Discussions

Let me know what works for you. Could be a quick fix

<img width="1671" height="831" alt="Image" src="https://github.com/user-attachments/assets/5ff9ae83-53f7-4e52-af96-200a74484358" />

**Comment by DavidVadnais** [comment]
## I pulled my users for some feedback:

“I don't think it makes sense to have it (ticket description) in ‘followup’.  That's me.” - @jhohum

"Agreed.  I think it's pretty obvious.  My guess is that it ended up in "followup" because that was easy from the original coding standpoint, not because it was good for UX." - @efiring

"My own preference is for the description to be a slight elaboration of what is in the title, but the long description should be in the followups.  So if I create a ticket from someone's long email, rather than forward it and get a novel in the description, I'll summarize and put the novel as the first followup.   ... if I have time.  I've had emergency/lazy moments when I did just forward the email.  It's awfully convenient." - @jhohum  

"While we're at it, I never liked how a ticket created by an incoming email fills the description." - Drew

## My opinion 
I personally also think having the ticket description in its own location makes sense. 

The originators of this issue talk about how the file attachments are tied to the follow-ups in the database. Do you have an idea for that? Often people will have attachments to their email when they reach out to a helpdesk.
